{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -11,17 +11,35 @@
 """
 
 import os
+import io
 import environ
 import logging
+import google.auth
+from google.cloud import secretmanager
+from google.auth.exceptions import DefaultCredentialsError
+from google.api_core.exceptions import PermissionDenied
 from modules.manifest import get_modules
 
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+env_file = os.path.join(BASE_DIR, ".env")
 env = environ.Env()
+env.read_env(env_file)
```

Comment:
Why is this necessary? `environ.Env()` should load `.env` automatically if it exists.

Reply:
`environ,Env()` this doesn't load .env file, we have to call read_env to load from file. In our case it's because of  `pipenv`, `pipenv shell` this loads if there is .env file.

Reply:
You're right. I had checked the docs and misread them.